src/config.js

```diff
@@ -44,6 +44,12 @@ const config = {
     decodeHTTP: (process.env.EPSAGON_DECODE_HTTP || 'TRUE').toUpperCase() === 'TRUE',
     disableHttpResponseBodyCapture: (process.env.EPSAGON_DISABLE_HTTP_RESPONSE || '').toUpperCase() === 'TRUE',
     loggingTracingEnabled: (process.env.EPSAGON_LOGGING_TRACING_ENABLED || (!utils.isLambdaEnv).toString()).toUpperCase() === 'TRUE',
+    sendBatch: (process.env.EPSAGON_SEND_BATCH || 'FALSE').toUpperCase() === 'TRUE',
+    batchSize: (Number(process.env.EPSAGON_BATCH_SIZE) || consts.DEFAULT_BATCH_SIZE),
```

Comment:
We should have an upper limit on batch sizes (that should probably match the ones that will be configured in TC)

Reply:
So you say the only limit for batch size will be a constant? not user configured?

Reply:
I'll define softLimit (user defined) + hardLimit (equal to TC limit)

Reply:
👍